Source/Diagnostics/ReducedDiags/BeamRelevant.cpp

```diff
@@ -37,7 +37,7 @@ BeamRelevant::BeamRelevant (std::string rd_name)
     //       13: rms gamma
     // 14,15,16: emittance x,y,z
     //       17: charge
-    m_data.resize(18,0.0);
+    m_data.resize(18, amrex::Real(0.0));
```

Comment:
You should be able to do this, if you want to be more compact
```suggestion
    m_data.resize(18, 0.0_rt);
```